engine/src/main/java/module-info.yml

```diff
@@ -0,0 +1,21 @@
+name: org.hibernate.search.engine
+
+requires:
+  - module: org.jboss.logging
+
+  - module: org.hibernate.search.util.common
+    transitive: true
+
+uses:
+  - org.hibernate.search.engine.environment.bean.spi.BeanConfigurer
+
+exports:
+  - package: org\.hibernate\.search(\..+)*\.(logging|reporting)\.impl
+    pattern: true
+    to:
+      - org.jboss.logging
+      - org.hibernate.search.engine
+  - package: org\.hibernate\.search(\..+)*\.impl
+    pattern: true
+    to:
+      - org.hibernate.search.engine
```

Comment:
Do we have to repeat these for every module? Or is there some concept of includes that we could leverage?

Reply:
I haven't seen anything in the docs directly. But it is using `snakeyaml` so if that one can understand some sort of includes we can give that a try